pkg/config/configuration.go

```diff
@@ -43,6 +43,7 @@ const (
 	SpiffeIDPrefix      = "spiffe://"
 	HTTPProtocol        = "http"
 	GRPCProtocol        = "grpc"
+	ActorRentrancy      = "Actor.Reentrancy"
```

Comment:
Please, create some sort of enum with features and make them strongly typed. For example: https://www.ribice.ba/golang-enums/